Source/OCMock/OCMBoxedReturnValueProvider.m

```diff
@@ -18,6 +18,18 @@
 #import "OCMFunctionsPrivate.h"
 #import "NSValue+OCMAdditions.h"
 
+static BOOL IsZeroBuffer(const char* buffer, size_t length)
+{
+    for(size_t i = 0; i < length; ++i)
```

Comment:
Why is this ++i, and not i++?

Reply:
I didn't know that there was a preference for one over the other. Is there a reason for i++?

Reply:
Not really. I guess decades of habit on my side.